appveyor.yml

```diff
@@ -1,9 +1,11 @@
 version: 1.0.{build}
 
+# Operating system (build VM template)
+os: Visual Studio 2015 CTP
+
 environment:
   matrix:
-#   - VS_VERSION: MSVC2012
-    - VS_VERSION: MSVC2013
```

Comment:
This is not how it is done. You would have to crete a whole environments (see the `os` entry above)